src/popup/router/components/PanelItem.vue

```diff
@@ -74,6 +74,7 @@ export default {
   .panel-item-info {
     color: #ccc;
     font-weight: normal;
+    line-height: normal;
```

Comment:
Don't see a reason to add this.

Reply:
| With | Without |
|---|---|
| <img width="406" alt="Screenshot 2020-11-17 at 14 33 27" src="https://user-images.githubusercontent.com/9007851/99385318-de769c80-28e1-11eb-8b17-c378309c730a.png"> | <img width="406" alt="Screenshot 2020-11-17 at 14 33 23" src="https://user-images.githubusercontent.com/9007851/99385329-e1718d00-28e1-11eb-8f4b-67b9449ad5ff.png"> |

To fir to some phone screen?🤔

Reply:
There are different line-height on button (Remove account) and a link (Permissions settings and upper panels). Browser set it by default for buttons and it rewrites our line height defined for html in https://github.com/aeternity/superhero-wallet/blob/develop/src/common/base.scss#L21, but for link it's not, so line-height became 1.5

Reply:
We anyway will have a different design for this page very soon, so I don't think we have to spend time refactoring it now.

Reply:
@mradkov So, remove this line?